app/boards/arm/adv360pro/adv360pro.dtsi

```diff
@@ -44,7 +44,7 @@
 
         >;
     };
-    ext-power {
+    EXT_POWER {
```

Comment:
Might be helpful to include the
```c
// Node name must match original "EXT_POWER" label to preserve user settings.
```
comment that's used in other shields.

Reply:
Added